.github/workflows/release.yml

```diff
@@ -25,3 +25,39 @@ jobs:
         run: |
           python3 -m build --sdist --wheel .
           twine upload dist/*
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+          pipenv install --dev --system --skip-lock
+          pip install https://github.com/Toloka/stubmaker/archive/refs/heads/main.zip
+      - name: Checkout docs repository
+        uses: actions/checkout@v3
+        with:
+          repository: 'Toloka/docs'
```

Comment:
We use `"` instead of `'` in YAML, when possible. Could you please replace them accordingly in the entire file? The only exception, I think, is the `if:` block that accepts only `'`.